stdlib.sh

```diff
@@ -1567,16 +2191,31 @@ function __STDLIB_API_1_std::capture() { # {{{
 			;;
 	esac
 
-	if ! type -t "${cmd:-}" >/dev/null; then
-		error "Invalid parameters: <command> '${cmd:-}' not found"
-		std_ERRNO=$( errsymbol EARGS )
-		return 1
+	## NB: 'type' check requires that args are passed separately from the cmd
+	##     to be executed. This may render specifying subcommands complex,
+	##     especially when one wants to prepend them with env variables like
+	##     DEBUG=1. Therefore, we can either pass cmd and args separately and
+	##     use the old logic to check availability of the command being invoked,
+	##     or pass the subcommand as a single param and skip the check.
+	if [[ ! -z "${args}" ]]; then
```

Comment:
It's your project therefore you decide the style, no issues here.

I read somewhere that you should prefer "! -z" over "-n" because it's more "reliable". But it was long ago and I forgot the details. And maybe it was to prefer "! -n" over "-z"...